fearless/Common/Model/ChainRegistry/AssetModel.swift

```diff
@@ -88,15 +93,7 @@ struct AssetModel: Equatable, Codable, Hashable {
 
     static func == (lhs: AssetModel, rhs: AssetModel) -> Bool {
         lhs.id == rhs.id &&
-            lhs.chainId == rhs.chainId &&
-            lhs.precision == rhs.precision &&
-            lhs.icon == rhs.icon &&
-            lhs.priceId == rhs.priceId &&
-            lhs.symbol == rhs.symbol &&
-            lhs.type == rhs.type &&
-            lhs.transfersEnabled == rhs.transfersEnabled &&
-            lhs.currencyId == rhs.currencyId &&
-            lhs.displayName == rhs.displayName
+            lhs.chainId == rhs.chainId
```

Comment:
please put back removed comparsions, otherwise app will not handle changes in assets.json

Reply:
I'm resolved this. But the type is not compared, because type compared in ChainAssetModel